forgerock-auth/src/main/java/org/forgerock/android/auth/exception/ApiException.java

```diff
@@ -61,4 +61,12 @@ public String getError() {
         return error;
     }
 
+    @Override
+    public String toString() {
```

Comment:
my IDE warned to add this @NonNull, not a big deal...when i pulled this branch 

@Override
@NonNull
    public String toString()

Reply:
Convert to Kotlin class